hud-data/supercomphud-omega.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "SUPERCOMPHUD-OMEGA",
+  "author": "lavaOnoes",
+  "contributors": ["KACHOWHERO"],
+  "flags": [
+    "fonts",
+    "menus",
+    "scoreboards",
+    "materials"
+  ],
+  "repo": "https://github.com/lavaOnoes/SUPERCOMPHUD-OMEGA",
+  "hash": "89dc5548d68b4b5426799f0702878e8e9855044b",
+  "prerelease": false,
+  "resources": [
+    "supercomphud-main",
+    "1",
+    "counter",
+    "https://youtu.be/q8LgIzs_M2o"
```

Comment:
i'd remove this

Reply:
may i ask why? aren't these for the images i have in hud-resources to be displayed? unless im doing it wrong

Reply:
i think the video you included is not the one you intended lmao

Reply:
you'd be surprised